src/core/crypto/ed25519.ts

```diff
@@ -232,26 +192,93 @@ export class Ed25519PrivateKey extends PrivateKey {
     return new Ed25519PrivateKey(privateKey);
   }
 
-  static isPrivateKey(privateKey: PrivateKey): privateKey is Ed25519PrivateKey {
-    return privateKey instanceof Ed25519PrivateKey;
+  // endregion
```

Comment:
delete?

Reply:
the `// endregion` comment, togther with `// region Some descript` creates a custom foldable region in most recent editors

Reply:
<img width="774" alt="Screenshot 2024-02-12 at 3 06 35 PM" src="https://github.com/aptos-labs/aptos-ts-sdk/assets/7707719/bcc390f1-1c08-47b4-8f8a-584ebc58510b">